.github/workflows/celo-monorepo.yml

```diff
@@ -170,45 +170,25 @@ jobs:
           mkdir -p test-results/jest
           # Skipping packages that are tested in a specific job below
           yarn run lerna \
-            --ignore @celo/contractkit \
             --ignore @celo/protocol \
             --ignore @celo/celotool \
-            --ignore @celo/celocli \
             --ignore @celo/env-tests \
-            --ignore '@celo/wallet-*' \
             run test
       - name: Upload Jest Test Results
         uses: actions/upload-artifact@v3
         with:
           name: Jest Test Results
           path: test-results/jest
-  wallet-test:
-    name: Wallet test
-    runs-on: ['self-hosted', 'monorepo-node18']
-    timeout-minutes: 30
-    needs: install-dependencies
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: recursive
-      - name: Sync workspace
-        uses: ./.github/actions/sync-workspace
-        with:
-          artifacts_to_cache: ${{ needs.install-dependencies.outputs.artifacts_to_cache }}
-      - name: Run Wallet tests
-        run: |
-          yarn run lerna --scope '@celo/wallet-*' run test
+
 
   protocol-test-release:
     name: Protocol Test Release
     runs-on: ['self-hosted', 'monorepo-node18']
     timeout-minutes: 500
     needs: [install-dependencies, lint-checks]
     if: |
-      github.base_ref == 'master' || contains(github.base_ref, 'staging') || contains(github.base_ref, 'production') ||
+      github.base_ref == 'master' || contains(github.base_ref, 'release') || contains(github.base_ref, 'production') ||
```

Comment:
this is additional. we dont use git branches with staging in the branch. but we do with release